simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/component/variable/VariableAccessor.java

```diff
@@ -59,6 +61,10 @@ public Station getStation() {
 
     public abstract Map<AttributeType, SetVariableValidator> getVariableValidators();
 
+    public abstract List<ReportDatum> generateReportData(String componentName);
```

Comment:
Consider having non-abstract method with NOP.

Reply:
then you would just have an unused variable in the method in the abstract class.

i actually want this to be abstract because I don't want people to forget to implement this method when adding new variable accessors.